app/src/main/java/com/orange/ouds/app/ui/tokens/TokenCategoryDetailScreen.kt

```diff
@@ -121,14 +125,14 @@ fun TokenCategoryDetailScreen(tokenCategory: TokenCategory, onSubcategoryClick:
                                 Column(
                                     modifier = Modifier
                                         .weight(1f)
-                                        .padding(start = if (isTypographyProperty) OudsSpaceKeyToken.Fixed.None.value else OudsSpaceKeyToken.Fixed.Medium.value)
+                                        .padding(start = if (noIllustrationProperty) OudsSpaceKeyToken.Fixed.None.value else OudsSpaceKeyToken.Fixed.Medium.value)
```

Comment:
It's also possible to remove the padding here and use the `horizontalArrangement` parameter of the `Row` to achieve the same effect:

```kotlin
horizontalArrangement = Arrangement.spacedBy(OudsSpaceKeyToken.Fixed.Medium.value)
```

This allows to get rid of the `noIllustrationProperty` variable too.